.github/actions/test-khiops-install/action.yml

```diff
@@ -1,74 +1,57 @@
 ---
 name: Test Khiops Installation
```

Comment:
Perhaps we can specify the "default" shell once and for all? Something like (see also https://github.com/actions/runner/blob/main/docs/adrs/0549-composite-run-steps.md#shell-and-working-directory):

```
inputs:
  default_shell:
    description: 'Bash'
    default: 'bash'
```

And then in each step:

```
shell: ${{ inputs.default_shell }}
```

Thus, we set `bash` only once and we refer to it in each step?

Reply:
I think it adds complexity. For me it is not a problem to set the shell at each step...

Reply:
OK, fair enough.